gslib/utils/copy_helper.py

```diff
@@ -3472,13 +3489,14 @@ def GetSourceFieldsNeededForCopy(dst_is_cloud,
         'mediaLink',
         'metadata',
         'metageneration',
-        'size',
         'storageClass',
         'timeCreated',
     ])
     # We only need the ACL if we're going to preserve it.
     if preserve_acl:
       src_obj_fields_set.update(['acl'])
+    if not file_size_will_change:
```

Comment:
Is there any harm if the size is added even for stet? I was wondering if we can avoid having to check for this case and keep it generic enough.

Reply:
The changing file size results in things like "Copied 1.2/1.0GiB" if we leave seek-ahead iterator as it is.

Reply:
Ack